blog/2022/10/22/vm-sealos-creat.md

```diff
@@ -1,11 +1,11 @@
-## 基于multipass使用脚本一键启动一主一从k8s集群
-
-
-#### [multipass配置教程](https://laogen.cc/multipass-bridge-network-on-linux/)
-
-注意：这个教程里面创建的bridges叫br0，而脚本里面的默认bridge是br0-test，可以配置桥接的时候设置成br0-test。
-
+---
+slug: 基于sealos使用脚本一键启动一主一从k8s集群
+title: 基于sealos使用脚本一键启动一主一从k8s集群
```

Comment:
Title and slug please use en, and the filename is missing 'e': "create"

Reply:
fix it